lighthouse-core/lib/log.js

```diff
@@ -40,8 +40,9 @@ class Emitter extends EventEmitter {
    * Fires off all status updates. Listen with
    * `require('lib/log').events.addListener('status', callback)`
    * @param {string} title
+   * @param {...*} args
    */
-  issueStatus(title, args) {
+  issueStatus(title, ...args) {
     if (title === 'status' || title === 'statusEnd') {
       this.emit(title, args);
```

Comment:
yes. Kind of confusingly `args` was previously an array, and event consumers are expecting a single array from the event, but that made it awkward to call when switching everything else to rest params, so it was easiest to just have varargs and group them up into an array here.